src/microbe_stage/organelle_components/MovementComponent.cs

```diff
@@ -49,6 +48,15 @@ protected override void CustomAttach()
         SetSpeedFactor(0.25f);
     }
 
+    protected override bool NeedsUpdateAnyway()
+    {
+        // The basis of the transform represents the rotation, as long as the rotation is not modified,
+        // the organnel needs to be updated.
+        // TO DO:
+        // Calculated rotations should never equal the identity, it should be kept an eye on if it does
```

Comment:
Also this TODO needs a github issue opened for this and linked here

Reply:
So the issue is?

Reply:
This needs to be like this:
```
TODO: Calculated rotations should never equal the identity, it should be kept an eye on if it does
See: CREATE A NEW ISSUE AND LINK IT HERE
```
This is presumably here due to an engine bug? Could you clarify that a bit here in this comment?

Reply:
Is there a reason to link it if its Godot related tho?

Reply:
I've created issues in our repo to track issues that are Godot problems.